com.unity.sg2/Editor/GraphUI/Importers/ShaderGraphAsset.cs

```diff
@@ -44,8 +44,8 @@ public static GraphHandler CreateBlankGraphHandler()
             var contextKey = Registry.ResolveKey<ShaderGraphContext>();
             var propertyKey = Registry.ResolveKey<PropertyContext>();
             GraphHandler graph = new ();
-            graph.AddContextNode(contextKey, defaultRegistry);
             graph.AddContextNode(propertyKey, defaultRegistry);
+            graph.AddContextNode(contextKey, defaultRegistry);
```

Comment:
currently, we always add the context to the end of the list. We should probably change that. Temporary fix